app/assets/javascripts/components/hamburger_menu.jsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { slide as Menu } from 'react-burger-menu';
+import CustomLink from './CustomLink.jsx';
+
+const HamburgerMenu = React.createClass({
+  propTypes: {
+    rootUrl: React.PropTypes.string,
+    logoPath: React.PropTypes.string,
+    fluid: React.PropTypes.bool,
+    exploreclass: React.PropTypes.string,
```

Comment:
All these props should ideally be the standard javaScriptCase, with first letters of words after the first capitalized.